### ISSUE TITLE
Quest: The Warm-Up

### ISSUE THREAD

**Issue by Hokraa** [dalaranwow/dalaran-wow#5529]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who) - I do not get credit for killing Kirgaraak in Storm Peaks.  

**Expected behaviour**: Tell us what should happen instead - I should get credit for killing Kirgaraak when he is killed by my actions in a fair fight.

**Steps to reproduce the problem**:

1. I engaged Kirgaraak on the war mount, used the special attacks, my mount died, and I finished him off with regular attacks.
2. I tried it again, using only special attacks on the war mount, got Kirgaraak down to about 1000 hp, then he bugged out, went back to full health and killed me.
3. Tried it again, no credit for quest.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
